ratbag_emu/__init__.py

```diff
@@ -5,3 +5,7 @@
 from .endpoint import Endpoint  # noqa: 401
 from .firmware import Firmware  # noqa: 401
 from .hw_component import HWComponent  # noqa: 401
+
+import logging
+__logger = logging.getLogger('ratbag-emu')
+__logger.setLevel(logging.INFO)
```

Comment:
Why is this needed? It can be manually enabled. pytest can also manually enable it via cli options.

Reply:
right. But then can we turn this on in the CI? I mean, pytest will hide logs when everything is fine, so better at least have `--log-level=INFO` by default

Reply:
Yes we can. IIRC pytest will output all the logs at the end, if you want to have them at runtime you could do `--log-file=/dev/stdout`. You can also just save the logs to a normal file.

I am not sure we want to enable this by default in the module. I am fine with enabling them by default in the tests if you want.

Reply:
>  I am fine with enabling them by default in the tests if you want.

OK done: 249a71e46618..7c885bef8d5e  master -> master (in libratbag-tests)